lib/test_transform.dart

```diff
@@ -91,15 +99,14 @@ class TestTransform implements Transform {
     if (package == id.package) {
       path = id.path;
     } else {
+      if (packageResolver == null) await _setupPackageResolver();
       String pathWithoutLib =
           id.path.split('/').skip(1).join(io.Platform.pathSeparator);
-      path = [packageRoot, id.package, pathWithoutLib]
-          .join(io.Platform.pathSeparator);
+      path = (await packageResolver.urlFor(id.package, pathWithoutLib))
+          .toFilePath();
     }
     io.File file = new io.File(path);
-    if (!(await file.exists())) {
-      throw new AssetNotFoundException(id);
-    }
+    if (!(await file.exists())) throw new AssetNotFoundException(id);
```

Comment:
Is this an intended change?

Reply:
Yes, I just didn't see the need to keep the block.